plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -1820,6 +1820,65 @@ public void testTableWithTrailingSlashLocation(boolean partitioned)
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test(dataProvider = "deleteFiltersForTable")
+    public void testDeleteWithFilter(String createTableSql, String deleteFilter)
```

Comment:
Didn't we have tests that test deletion previously? If we did what is the point of these? They don't test the feature only correctness of deletion

Reply:
We may want to assert the plan like below:
```java
        assertUpdate(
                getSession(),
                format("DELETE FROM %s WHERE %s", table, deleteFilter),
                2,
                plan -> {
                    boolean tableDelete = searchFrom(plan.getRoot()).where(node -> node instanceof TableDeleteNode).matches();
                    assertTrue(tableDelete, "...");}
        );
```